cmd/wails/0_setup.go

```diff
@@ -91,19 +91,52 @@ func checkLibraries() (errors bool, err error) {
 		// Check library prerequisites
 		requiredLibraries, err := cmd.GetRequiredLibraries()
 		if err != nil {
-			return false, err
+			return true, err
```

Comment:
so after fixing this bug too, arch still doesn't run the prerequisites tests.

Reply:
That's so weird. I should get time this weekend to debug this. Will have to debug in the container 📦